qan-api2/main.go

```diff
@@ -141,7 +141,7 @@ func runJSONServer(ctx context.Context, grpcBindF, jsonBindF string) {
 	marshaller := &grpc_gateway.JSONPb{
 		MarshalOptions: protojson.MarshalOptions{ //nolint:exhaustivestruct
 			UseEnumNumbers:  false,
-			EmitUnpopulated: false,
+			EmitUnpopulated: true,
```

Comment:
I'm not sure if we need this for QA, do we?

Reply:
I don't think so.

Reply:
let's enable anyway to have consistent API